L1Trigger/TrackerTFP/src/LayerEncoding.cc

```diff
@@ -0,0 +1,117 @@
+#include "L1Trigger/TrackerTFP/interface/LayerEncoding.h"
+#include "L1Trigger/TrackerDTC/interface/SensorModule.h"
```

Comment:
1) Is it intentional that a class named LayerEncoding should exist both in TrackerTFP/ and in TrackerDTC/
2) The one in TrackerDTC/ does not compile, because it tries including the non-existant file L1Trigger/TrackerDTC/interface/SensorModule.h . A file of this name does exist in L1Trigger/TrackTrigger/ .

Reply:
I see this is my code:
https://github.com/cms-L1TK/cmssw/blob/tschuh12/L1Trigger/TrackerDTC/src/LayerEncoding.cc#L1-L2
When I shall rename a class please let me know to what.

Reply:
thanks, fixed it. When I shall change a name of a class please let me know to what.

Reply:
The two versions of LayerEncoding are protected by different namespaces, so should be OK. You are best placed to judge if they deserve different names.